Formula/py-spy.rb

```diff
@@ -0,0 +1,27 @@
+class PySpy < Formula
+  desc "Sampling profiler for Python programs"
+  homepage "https://github.com/benfred/py-spy"
+  url "https://github.com/benfred/py-spy/archive/refs/tags/v0.3.7.tar.gz"
+  sha256 "5a75df5dd1ed683b80bb6a89007b4fe9f7f0a5129cf613ff70e94029d11e87a5"
+  license "MIT"
+  head "https://github.com/benfred/py-spy.git"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  def caveats
+    <<~EOS
+      py-spy requires root privileges so you will need to run
+      `sudo py-spy`.
+      You should be certain that you trust any software you grant root privileges.
+    EOS
+  end
+
+  test do
+    output = shell_output("#{bin}/py-spy --help 2>&1")
```

Comment:
Not sure how to do test with sudo, so l've opted to do just this.

Reply:
Try running it without sudo and trying to `assert_match` the error message.